browser/resources/settings/brave_overrides/settings_menu.js

```diff
@@ -99,10 +101,26 @@ RegisterStyleOverride(
         height: 20px;
       }
 
+      a[href].iron-selected::before {
+        content: "";
+        position: absolute;
+        top: 50%;
+        left: -30px;
+        transform: translateY(-50%);
+        display: block;
+        height: 34px;
```

Comment:
```suggestion
        height: 170%;
```

@petemill, this could solve your concern as the element height is 20px, and the design calls for the accent line to be 34px.